proving/proving_test.go

```diff
@@ -16,6 +16,8 @@ import (
 
 func getTestConfig(tb testing.TB) (config.Config, config.InitOpts) {
 	cfg := config.DefaultConfig()
+	cfg.K2PowDifficulty = 0x00FFFFFF_FFFFFFFF
+	cfg.K3PowDifficulty = 0x00FFFFFF_FFFFFFFF
```

Comment:
Could we make these the default values instead of `math.MaxUint64`. Maybe `0x0FFFFFFF_FFFFFFFF` is a better default, since on average a node would only need to try 16 values but then we would also have this in system tests, otherwise the systemtests would not do PoW on k2 and k3